lb_content_resolver/model/directory.py

```diff
@@ -0,0 +1,18 @@
+from peewee import *
+from lb_content_resolver.model.database import db
+
+
+class Directory(Model):
+    """
+    Basic metadata information about a recording on disk (a track).
+    """
+
+    class Meta:
+        database = db
+
+    id = AutoField()
+    dir_path = TextField(null=False, unique=True)
```

Comment:
Add index=True here, since you are querying by dir_path.

Reply:
Since it's unique, index is automatically created. We don't have it for https://github.com/metabrainz/listenbrainz-content-resolver/blob/38bafae5071e8ccf3a6d5cfa1a424a8db5855938/lb_content_resolver/model/recording.py#L15 either.

See https://docs.peewee-orm.com/en/latest/peewee/models.html#single-column-indexes-and-constraints